app/main/views/suppliers.py

```diff
@@ -315,9 +316,12 @@ def edit_supplier_duns_number(supplier_id):
 @main.route('/suppliers/<int:supplier_id>/edit/declarations/<string:framework_slug>', methods=['GET'])
 @role_required('admin-ccs-sourcing')
 def view_supplier_declaration(supplier_id, framework_slug):
+    if framework_slug in DEPRECATED_FRAMEWORK_SLUGS:
+        abort(404)
```

Comment:
Yep, I think this is fine. If admins really want to see g-cloud-4 details they're probably better off looking at a bit of paper somewhere...